addons/account_peppol/wizard/peppol_config_wizard.py

```diff
@@ -0,0 +1,158 @@
+    account_peppol_migration_key = fields.Char(related='company_id.account_peppol_migration_key', readonly=False)
+
+    service_json = fields.Json(
+        compute='_compute_service_json',
+        store=True,
+        readonly=False,
+        help="JSON representation of peppol services as retrieved from the peppol server.",
+    )
+    service_info = fields.Html(compute='_compute_service_info')
+    service_ids = fields.One2many(
+        comodel_name='account_peppol.service',
+        inverse_name='wizard_id',
+        compute='_compute_service_ids',
+        store=True,
+        readonly=False,
+    )
+
+    # -------------------------------------------------------------------------
+    # COMPUTES
+    # -------------------------------------------------------------------------
+
+    @api.depends('account_peppol_proxy_state')
+    def _compute_service_json(self):
+        for wizard in self:
+            if wizard.account_peppol_proxy_state == 'receiver':
+                wizard.service_json = wizard.account_peppol_edi_user._peppol_get_services().get('services')
+            else:
+                wizard.service_json = False
+
+    @api.depends('account_peppol_proxy_state')
+    def _compute_service_info(self):
+        for wizard in self:
+            message = ''
+            if wizard.account_peppol_proxy_state == 'receiver':
+                supported_doctypes = self.env['res.company']._peppol_supported_document_types()
+                if (non_configurable := [
+                    identifier
+                    for identifier in (wizard.service_json or {})
+                    if identifier not in supported_doctypes
+                ]):
+                    message = Markup('%s<ul>%s</ul>') % (
+                        _(
+                            "The following services are listed on your participant but cannot be configured here. "
+                            "If you wish to configure them differently, please contact support."
+                        ),
+                        Markup().join(
+                            Markup('<li>%s</li>') % (wizard.service_json[identifier]['document_name'])
+                            for identifier in non_configurable
+                        ),
+                    )
+            wizard.service_info = message
+
+    @api.depends('account_peppol_proxy_state')
+    def _compute_service_ids(self):
+        """Get the selectable document types.
+
+        Synthesize a combination of locally available document types and those added to the user on
+        the IAP, add the relevant services.
+        """
+        supported_doctypes = self.env['res.company']._peppol_supported_document_types()
+        for wizard in self:
+            if wizard.account_peppol_proxy_state == 'receiver':
+                wizard.service_ids = self.env['account_peppol.service'].create([
```

Comment:
:grimacing: You must not create anything in a compute! Use the `Command.create` instead